src/modules/menu/i18n/menu.provider.js

```diff
@@ -4,13 +4,49 @@ export const provide = (lang) => {
 		case 'en':
 			return {
 				//the first part of the snake_case key should be the name of the related module
-				menu_main_open_button: 'Open menu'
+				menu_main_open_button: 'Open Menu',
+				misc_content_panel_settings: 'Appearance',
+				misc_content_panel_dark_mode: 'Dark theme',
+				misc_content_panel_fullscreen: 'Fullscreen',
+				misc_content_panel_language: 'Language',
+				misc_content_panel_information: 'Information',
+				misc_content_panel_help: 'Help',
+				misc_content_panel_Contact: 'Contact',
+				misc_content_panel_github: 'GitHub',
+				misc_content_panel_terms_of_use: 'Terms of Use',
+				misc_content_panel_privacy_policy: 'Privacy Policy',
+				misc_content_panel_imprint: 'Imprint',
+				misc_content_panel_more_links: 'More Applications',
+				misc_content_panel_gdo_header: 'Geodaten Online',
+				misc_content_panel_gdo_text: 'Order digital data online and download them immediately.',
+				misc_content_panel_gp_header: 'Geoportal Bayern',
+				misc_content_panel_gp_text: 'Central access to geospatial data and services.',
+				misc_content_panel_ea_header: 'Energie-Atlas Bayern',
+				misc_content_panel_ea_text: 'Central portal for saving energy, energy efficiency, and renewable energies.'
```

Comment:
sorry, didn't see that earlier: all keys must meet our convention starting with the module name,
e.g. `misc_content_panel_settings` -> `menu_misc_content_panel_settings`